.github/workflows/integration.yml

```diff
@@ -0,0 +1,64 @@
+name: integration-test
+
+on:
+  schedule:
+    - cron: '0 0 * * 6'
```

Comment:
this test runs every saturday 00:00 ?

Reply:
That's right. This test runs every Saturday on the hour.
It seems to run at 10:00 Korean time.